test/browser_tests/shared_objects/video-player.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var _videoPlayer = element( by.css( '.video-player' ) );
+
+function _videoPlayerElement( selector ) {
+  return _videoPlayer.element( by.css( selector ) );
+}
+
+var VideoPlayer = {
+
+  videoPlayerCloseButton: _videoPlayerElement( '.video-player_close-btn' ),
+
+  videoPlayerVideoContainer:
+    _videoPlayerElement( '.video-player_video-container' ),
+
+  videoPlayerIframeContainer:
+    _videoPlayerElement( '.video-player_iframe-container' ),
+
+  getVideoPlayerIframe: function getVideoPlayerIframe( ) {
+    return _videoPlayerElement( '.video-player_iframe' );
+  },
+
+  videoPlayerImageContainer:
+    _videoPlayerElement( '.video-player_image-container' ),
+
+  videoPlayerPlayButton: _videoPlayerElement( '.video-player_play-btn' ),
+
+  videoPlayer: _videoPlayer
```

Comment:
Technically `_videoPlayer` wouldn't have an underscore prefix because it is getting exposed publicly here. (Versus `_videoPlayerElement`, which returns a value publicly but is not exposed itself publicly.)